example/pubspec.lock

```diff
@@ -35,7 +35,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "0.1.0+1"
+    version: "0.1.1+1"
```

Comment:
Why do you upgrade the version for this kind of update? We don't release for this kind of change that does not impact the user, right?

Reply:
The change was made automatically by my IDE/editor. The update was made to example project not the package. Also pubspec.lock file is used to synchronize package versions between developers who are working on the same project. Don't worry it doesn't impacts users. :)

Reply:
Ok, I don't' really understand how releases work for pub.dev and in the dart community! 😅 I will probably ask you questions directly on Slack 😄

Reply:
Ok, the changed file is lockfile used by dart's package manager (pub). It's same as package-lock.json / yarn.lock files in node.js / npm.

Reply:
Oh that's the file in the `example` folder! You told me but I did not get it! haha
Sorry for being a 🐌 today.